hasher-matcher-actioner/webapp/src/pages/settings/ThreatExchangeSettingsTab.jsx

```diff
@@ -70,16 +70,31 @@ export default function ThreatExchangeSettingsTab() {
         setShowToast(true);
       });
   };
+
+  const refreshDatasets = () => {
+    setLoading(false);
```

Comment:
hmm, don't we want the opposite? We are starting a request therefore loading is `true` or am I missing something?

Reply:
the name might be confusing, because it's equals to hidden, so when starting a request I set hidden as false

Reply:
then can we call it hidden or switch the conditional? Otherwise the PR looks good to me :)

Reply:
Thanks for pointing that out! I am still using the name loading and I've switched the conditional to ```hidden=!loading```, so it looks more logical